src/main/resources/openapi/transcriptions.yaml

```diff
@@ -535,6 +591,33 @@ components:
           example: 3
         workflow_comment:
           type: string
+
+    UpdateTranscriptionsRequest:
+      properties:
+        transcriptions:
```

Comment:
Don't think we need this transcriptions property in the Request Body - it should be fine to just accept an array of objects?

Reply:
A `transcriptions` array is what the LLD says...